package.json

```diff
@@ -61,7 +61,7 @@
       "eslint"
     ],
     "*.{ts,tsx}": [
-      "tsc --noEmit"
+      "tsc --noEmit --jsx react"
```

Comment:
Do we need the same flag in lint:js script?

Reply:
I don't think it's needed in the `lint:js`. The issue seems to only propagate itself when running in the incremental mode over the staged files.